operator/controllers/internal/csiscaleoperator/csiscaleoperator_package.go

```diff
@@ -682,3 +690,102 @@ func (s *CSIScaleOperator) GetNodeSelectors(nodeSelectorObj []v1.CSINodeSelector
 
 	return nodeSelectors
 }
+
+func (s *CSIScaleOperator) GetPodAntiAffinity(controllerName string) *corev1.PodAntiAffinity {
+
+	appName := ""
+	switch controllerName {
+	case "attacher":
+		appName = config.GetNameForResource(config.CSIControllerAttacher, s.Name)
+	case "provisioner":
```

Comment:
I see antiaffinity cases other than attacher are unused, do we want to keep these commented if needed in future, or remove otherwise?

Reply:
the antiaffinity for other sidecars are missing (or not required)?

Reply:
as per latest discussion, for now only attacher replicas are set to be 2, so podAntiAffinity has to be set only for attacher. switch case can be removed from here.

Reply:
Ok

Reply:
Removed the switch case.